src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.ts

```diff
@@ -257,6 +255,7 @@ export class ExerciseDetailsStudentActionsComponent implements OnInit, OnChanges
             });
     }
 
+    // this method will be removed once text and modeling support new component
```

Comment:
Consider specifying this and similar comments as a TODO to highlight it and bring it to special attention